### PR TITLE
master-qa-60285-5 MFA sample source with new automated test

### PR DESCRIPTION
**Pull request by martamedio**
Rename from _POC_ to _Sample_, maintaining the structure that we use in the rest of the MFA modules.

@gaborlovasdotcom please, check my [test changes](https://github.com/liferay-appsec/liferay-portal/commit/dac6eac92bf201f3c2eb1a5fc302bc9b3a557f1f) before forward this PR

cc/ @csierra

### CODE REVIEW

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/java/com/liferay/multi/factor/authentication/sample/internal/checker/SampleBrowserMFAChecker.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/java/com/liferay/multi/factor/authentication/sample/internal/checker/SampleBrowserMFAChecker.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/java/com/liferay/multi/factor/authentication/sample/internal/settings/definition/MFASampleCompanyConfigurationBeanDeclaration.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/**
```
Comment (liferay-appsec) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/java/com/liferay/multi/factor/authentication/sample/internal/settings/definition/MFASampleCompanyConfigurationBeanDeclaration.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/resources/META-INF/resources/mfa_sample_checker/verify_browser.jsp [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<h3>
+	<liferay-ui:message key="please-enter-something" />:
+</h3>
+
+<aui:input label="mfa-sample" name="mfaSample" showRequiredLabel="yes" />
```
Comment (liferay-appsec) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/resources/META-INF/resources/mfa_sample_checker/verify_browser.jsp:23]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -0,0 +1,4 @@
+com.liferay.multi.factor.authentication.sample.internal.checker.SampleBrowserMFAChecker=Multi-Factor Authentication Sample
```
Comment (liferay-appsec) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-sample/src/main/resources/content/Language.properties:1]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by martamedio** [comment]
ci:test:sf

**Comment by martamedio** [comment]
ci:test:relevant

**Comment by martamedio** [comment]
ci:forward

**Pull request closed without merging**